dev-docs/modules/mediafilterRtdProvider.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page_v2
+title: The Media Trust Real-Time Ad Defense
+display_name: The Media Trust Real-Time Ad Defense
+description: Stop malware, redirects, scams, offensive ad content, heavy ads, and more from ruining your audience's experience.
+page_type: module
+module_type: rtd
+module_code : mediafilterRtdProvider
+enable_download : true
+vendor_specific: true
+sidebarType : 1
+---
+
+# The Media Trust Real-Time Ad Defense
+
+## Overview
+
+ As malvertising, scams, and controversial and offensive ad content proliferate across the digital media ecosystem, publishers need advanced controls to both shield audiences from malware attacks and ensure quality site experience. With the market’s fastest and most comprehensive real-time ad quality tool, The Media Trust empowers publisher Ad/Revenue Operations teams to block a wide range of malware, high-risk ad platforms, heavy ads, ads with sensitive or objectionable content, and custom lists (e.g., competitors). Customizable replacement code calls for a new ad to ensure impressions are still monetized.
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/VBHRiirge7s/0.jpg)](https://mediatrust.com/how-we-help/media-filter "The Media Filter")
```

Comment:
FWIW, I need to see how this winds up, but we've never had an adapter add an image. It might be ok if it's properly labelled. e.g. "Watch our youtube video for more info". Also, I don't believe this is the right way to embed images. 
```
![Prebid Server Web Banner Architecture](/assets/images/prebid-server/pbs-js-banner-architecture.png){:class="pb-xlg-img"}
```
Also, please make sure you load assets with HTTPS

Reply:
We removed the image.